serpentTools/parsers/sensitivity.py

```diff
@@ -331,6 +331,10 @@ def plot(self, resp, zai=None, pert=None, mat=None, sigma=3,
         mat: None or str or list of strings
             Plot sensitivities due to these materials. Passing ``None``
             will plot against all materials.
+        mevscale: bool, optional
+            Flag for plotting energy grid in MeV units. Default is ``False``.
+        egrid : numpy.array, optional
+            Energy grid for sampling reaction rates ratios. Default is ``None``.
```

Comment:
I would recommend adding a remark on how these will be used to update the plots. At first read, this may come across as the reaction rate ratios will be plotted against these energies. Additionally, this applies to all sensitivities, not just the reaction rate ratios